packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -64,6 +64,28 @@ function parseNumber(
   }
 }
 
+/**
+ * Parses a LogLevelString as a LogLevel
+ * @param key
+ * @param environment
+ * @param values
+ */
+function parseLogLevel(
+  key: keyof ENVIRONMENT,
+  environment: ENVIRONMENT_MAP | ENVIRONMENT,
+  values: ENVIRONMENT_MAP
+) {
+  const value = values[key];
+  const logLevelStrings = Object.keys(LogLevel).filter(
+    k => isNaN(Number(k)) === true
+  ) as LogLevelString[];
+  const regex = new RegExp(logLevelStrings.join('|'), 'i');
+  if (value && typeof value === 'string' && regex.test(value)) {
```

Comment:
@naseemkullah would that work here?

```suggestion
  const regex = new RegExp(logLevelStrings.join('|'), 'i');
  if (value && typeof value === 'string' && logLevelStrings.includes(value)) {
```

Reply:
Probably @iliasbhal but I just changed the function to a simpler switch case 😅